client/post-editor/editor-revisions-list/header.jsx

```diff
@@ -11,11 +11,13 @@ import { localize } from 'i18n-calypso';
 const EditorRevisionsListHeader = ( { numRevisions, translate } ) => {
 	return (
 		<div className="editor-revisions-list__header">
-			{ !! numRevisions &&
-				translate( '%(revisions)d revision', '%(revisions)d revisions', {
-					count: numRevisions,
-					args: { revisions: numRevisions },
-				} ) }
+			<span className="editor-revisions-list__count">
```

Comment:
Adding this span allows us to vertically centre the content of the header using flexbox. With the previous version, using `line-height`, there was a risk of the text rendering badly if it was ever long enough to wrap to the next line.